horace_core/sqw/file_io/@faccess_sqw_v4/get_sqw.m

```diff
@@ -68,16 +68,25 @@
 [obj,sqw_skel] = obj.get_all_blocks(sqw_skel,'ignore_blocks',skip_blocks);
 
 if ~(opts.head || opts.his)
+    detpar = sqw_skel.detpar;
+    if ~isempty(detpar)        
+        detpar = IX_detector_array(detpar);
+    else
+        detpar = IX_detector_array();
+    end
+    %detpar = repmat(detpar,numel(sqw_skel.experiment_info.expdata),1);
     sqw_skel.data = DnDBase.dnd(sqw_skel.data.metadata,sqw_skel.data.nd_data);
     sqw_skel.experiment_info = Experiment([],sqw_skel.experiment_info.instruments, ...
         sqw_skel.experiment_info.samples,sqw_skel.experiment_info.expdata);
+    sqw_skel.experiment_info.detector_arrays = ...
+        sqw_skel.experiment_info.detector_arrays.add_copies_(detpar, numel(sqw_skel.experiment_info.expdata));
 end
 
 
 % CRYSTAL ALIGNMENT FIXTURE: #TODO: #892 modify  and remove!
 proj = sqw_skel.data.proj;
 if isa(proj,'ortho_proj')
-    header_av = sqw_skel.experiment_info.header_average();
+    header_av = sqw_skel.experiment_info.header_average(sqw_skel.data);
     sqw_skel.data.proj = proj.set_ub_inv_compat(header_av.u_to_rlu(1:3,1:3));
 end
```

Comment:
why this is here? Its not in master and not in my branch. Its fixed and should not be here

Reply:
see my comment at 12.12pm